src/client/extension.ts

```diff
@@ -341,16 +352,19 @@ function handleWorkspaceFolderChange() {
     }
 }
 
-function showNotificationForCopilot(telemetry: TelemetryReporter) {
+function showNotificationForCopilot(telemetry: TelemetryReporter, telemetryData:string, countOfActivePortals: string) {
```

Comment:
nit: rename `telemetryData` to `listOfOrgsJson` so it's clearer what is expected.